.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,76 @@
+name: Continuous integration
+on:
+  - push
+jobs:
+  build:
+    name: "JDK ${{ matrix.java }} on ${{ matrix.os }}"
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+            java: 8
+          - os: ubuntu-latest
+            java: 8
+          - os: ubuntu-latest
+            java: 11
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Check out NullAway sources
+        uses: actions/checkout@v2
+      - name: Cache Gradle caches
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-caches-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle-caches-
+      - name: Cache Gradle wrapper
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradlew-wrapper-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
+          restore-keys: ${{ runner.os }}-gradlew-wrapper-
+      - name: 'Set up JDK ${{ matrix.java }}'
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Build and test using Gradle and Java 8
+        run: ./gradlew verGJF build
+        if: matrix.java == '8'
+      - name: Build and test using Gradle and Java 11
+        run: ./gradlew :nullaway:test
+        if: matrix.java == '11'
+      - name: Report jacoco coverage
+        run: ./gradlew jacocoTestReport coveralls
+        if: runner.os == 'Linux' && matrix.java == '8'
+      - name: Check that Git tree is clean after build and test
+        run: ./.buildscript/check_git_clean.sh
+  publish_snapshot:
+    name: 'Publish snapshot'
+    needs: [build]
+    if: github.event_name == 'push' && github.repository == 'uber/NullAway' && github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out repository'
+        uses: actions/checkout@v2
+      - name: Cache Gradle caches
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-caches-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle-caches-
+      - name: Cache Gradle wrapper
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradlew-wrapper-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
+          restore-keys: ${{ runner.os }}-gradlew-wrapper-
+      - name: 'Set up JDK 8'
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: 'Publish'
+        env:
+          SONATYPE_NEXUS_USERNAME: ${{ secrets.SONATYPE_NEXUS_USERNAME }}
```

Comment:
For WALA I had to do this to get env vars to show up as Gradle properties in the build:

```
          ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: ${{ secrets.SONATYPE_NEXUS_USERNAME }}
          ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
```
But the WALA Gradle scripts are different from NullAway's so maybe this is correct here.

Reply:
Our scripts look for `SONATYPE_NEXUS_USERNAME`, etc, in env vars. So I suspect this will work, but let me test it.

Reply:
Actually, you were right, `ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME` works for some reason and `SONATYPE_NEXUS_USERNAME` doesn't. I could check out why, but since it seems to be working now, I'll just update the branch and land.

Reply:
It's because the Gradle scripts are looking for a project property `SONATYPE_NEXUS_USERNAME`, etc., and you need the `ORG_GRADLE_PROJECT_` prefix to have Gradle read the environment variable as a project property: https://docs.gradle.org/current/userguide/build_environment.html#sec:project_properties